Models/models.js

```diff
@@ -6,3 +6,17 @@ exports.selectTopics = () => {
     return result.rows;
   });
 };
+
+exports.selectArticles = (sort_by = "created_at") => {
+ 
+
+  const SQL = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, COUNT(articles.article_id) AS comment_count 
```

Comment:
could you write an additional test to make sure that the COUNT is getting the correct number of comments back from the db